src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Log.cs

```diff
@@ -8,6 +8,6 @@ namespace Microsoft.AspNetCore.Diagnostics;
 
 internal static partial class Log
 {
-    [LoggerMessage(0, LogLevel.Warning, "Enricher failed: {Enricher}.")]
+    [LoggerMessage(0, LogLevel.Warning, "Enricher failed: {Enricher}.")] 
```

Comment:
Using the following comment as a guidance https://github.com/dotnet/extensions/pull/4604#issuecomment-1790492949, I think that using Warning level is correct, because an enricher that throws is not a typical situation, and it would be good for a user to have visibility of error caused by enrichers.